content/docs/command-reference/remote/modify.md

```diff
@@ -553,22 +558,22 @@ for a full guide on using Google Drive as DVC remote storage.
   Also requires using `gdrive_client_secret`.
 
   ```dvc
-  $ dvc remote modify myremote gdrive_client_id 'client-id'
+  $ dvc remote modify --local myremote gdrive_client_id 'client-id'
   ```
 
 - `gdrive_client_secret` - Client secret for authentication with OAuth 2.0 when
   using a custom Google Client project. Also requires using `gdrive_client_id`.
 
   ```dvc
-  $ dvc remote modify myremote \
-        gdrive_client_secret 'client-secret'
+  $ dvc remote modify --local myremote \
+                      gdrive_client_secret 'client-secret'
   ```
 
 - `gdrive_user_credentials_file` - path where DVC stores OAuth credentials to
   access Google Drive data. `.dvc/tmp/gdrive-user-credentials.json` by default.
 
   ```dvc
-  $ dvc remote modify myremote \
+  $ dvc remote modify --local myremote \
         gdrive_user_credentials_file .dvc/tmp/mycredentials.json
   ```
```

Comment:
...and in all these

Reply:
this is not sensitive and could be shared in some cases (to use multiple GDrive remotes for example)

Reply:
OK. Reverted these 3.